### DISCUSSION TITLE
Seq Email Notifications with Seq.App.EmailPlus-Enhanced

### DISCUSSION THREAD

**Discussion by specialagentsoftware** [datalust/seq-tickets#2032] [General]
Hello, I was looking at setting up alerts that will generate email messages. On the form where you configure the alert shows a form field named Notification Message. It states that this is the message that will form the subject, title, or main body of the notification. I am wanting to specify the body here. Is there some specific syntax that is required to specify the body. So far whatever I put there is set as the subject and the property that I am specifying with handlebars {{}} syntax is not being replaced. I am trying to send custom body's with each different alert without modifying the default template since the default template works for all of the other alert messages. 

Any help would be appreciated.

**Comment by specialagentsoftware** [comment]
I am also trying to use the Additional Properties field to add the property that I Am wanting to use. I was not sure if that was the intended use of that field or not.

**Comment by specialagentsoftware** [comment]
I think I might have found something. I saw a control that allows for an override of the body template. When I selected that I got an option that would allow me to specify the body template on the alert. I am going to try that. Please let me know if that is not the way. Thank you.

**Comment by liammclennan** [comment]
Hi @specialagentsoftware 

This package (https://github.com/MattMofDoom/seq-app-htmlemail/tree/EnhancedEdition) is not published by Datalust. Our latest email apps are https://github.com/datalust/seq-app-mail
